src/content/en/updates/2017/09/autoplay-policy-changes.md

```diff
@@ -165,6 +167,46 @@ including Facebook, Instagram, Twitter, and YouTube.
       });
     </script>
 
+### WebAudio
+
+First, be reminded that it is good practise to wait for a user interaction
```

Comment:
practice, not practise (in this case it's a noun which has a 'c' even in the UK)

Reply:
Thanks!